src/store/traction/pacbio/runs/actions.js

```diff
@@ -1,36 +1,30 @@
 import handlePromise from '@/api/PromiseHelper'
 import * as PacbioRun from '@/api/PacbioRun'
-const pre_extension_time = 2
-const ccs_analysis_output = 'Yes'
-const loading_target_p1_plus_p2_default = 0.85
-
-const generateHiFiDefault = (systemName) => {
-  switch (systemName) {
-    case 'Sequel I':
-    case 'Sequel II':
-      return 'In SMRT Link'
-    case 'Sequel IIe':
-      return 'On Instrument'
-    default:
-      return ''
-  }
-}
 
 const splitPosition = (position) => {
   // match() returns [original, row, column] e.g "A10 => ["A10", "A", "10"]
   return position.match(/(\S)(\d+)/).slice(1)
 }
 
 const buildWell = ({ state }, position) => {
-  let generate_hifi = generateHiFiDefault(state.currentRun.system_name)
   let binding_kit_box_barcode = state.currentRun.default_binding_kit_box_barcode || ''
   let [row, column] = splitPosition(position)
+  state.currentRun.wellDefault.generate_hifi = PacbioRun.wellDefaults(
+    state.currentRun.system_name,
+  ).generate_hifi
+  const movie_time = state.currentRun.default_movie_time || state.currentRun.wellDefault.movie_time
+  const generate_hifi =
+    state.currentRun.default_generate_hifi || state.currentRun.wellDefault.generate_hifi
+  const ccs_analysis_output =
+    state.currentRun.default_ccs_analysis_output || state.currentRun.wellDefault.ccs_analysis_output
+  const pre_extension_time =
+    state.currentRun.default_pre_extension_time || state.currentRun.wellDefault.pre_extension_time
```

Comment:
I'm not quite sure why you have the fallback here, rather than just using the values under wellDefault. The advantage of that is this you can just spread wellDefault into the new object, rather than having to extract each value individually.